### PR TITLE
docs: readme asdf

### PR DESCRIPTION
**Pull request by dcdourado**
Hello guys,

This PR adds instructions and info of asdf to newcomers to the project.
Started this a while ago, and I'm not sure about the grouping comments, what's your opinion about it?

### CODE REVIEW

File: apps/authenticator/lib/authenticator.ex [hunk 1]
```diff
@@ -9,6 +9,10 @@ defmodule Authenticator do
   alias Authenticator.SignIn.Commands.{ClientCredentials, RefreshToken, ResourceOwner}
   alias Authenticator.SignOut.Commands.{SignOutAllSessions, SignOutSession}
 
+  ###############
+  # Sign In
+  ###############
```
Comment (dcdourado) [inline apps/authenticator/lib/authenticator.ex:14]: I'm referring to these

File: apps/authenticator/lib/authenticator.ex [hunk 2]
```diff
@@ -18,12 +22,20 @@ defmodule Authenticator do
   @doc "Delegates to #{ClientCredentials}.execute/1"
   defdelegate sign_in_client_credentials(input), to: ClientCredentials, as: :execute
 
+  ###############
+  # Session
+  ###############
```
Comment (dcdourado) [inline apps/authenticator/lib/authenticator.ex:27]: .

File: apps/authenticator/lib/authenticator.ex [hunk 3]
```diff
@@ -18,12 +22,20 @@ defmodule Authenticator do
...
+
   @doc "Delegates to #{GetSession}.execute/1"
   defdelegate get_session(input), to: GetSession, as: :execute
 
   @doc "Delegates to #{AccessToken}.verify_and_validate/1"
   defdelegate validate_access_token(token), to: AccessToken, as: :verify_and_validate
 
+  ###############
+  # Sign Out
+  ###############
```
Comment (dcdourado) [inline apps/authenticator/lib/authenticator.ex:37]: .

**Comment by lcpojr** [comment]
Could you please update this branch from master aggain?
I've fixed the CI crypto problem.

**Comment by dcdourado** [comment]
Hey @lcpojr can we remove the dialyzer step from the CI? It's really messing around.
I also would suggest removing `WIP — Ready for review`, as it doesn't do a lot...

**Comment by dcdourado** [comment]
> Hey @lcpojr can we remove the dialyzer step from the CI? It's really messing around.
> I also would suggest removing `WIP — Ready for review`, as it doesn't do a lot...

The cache is hitting but it still does find the `dialyzer.plt` file. Any thoughts?

**Comment by lcpojr** [comment]
I've finally fixed the dialyzer flow on CI.
It was not generating the PLT because of permissions, linux versions and so on ... but now it's fixed on master.

The `dialyzer` and `plt` folders has to exists in order to not compile every single commit all dependencies.

Sorry for all the trouble, i know it's a pain sometimes but it also helps a lot avoid mistakes with types and returns (i got some on my last pull request) and i think now that it's fixed we will more use of it.

**Comment by lcpojr** [comment]
@dcdourado try to update from master aggain.
It worked on my another PR.

**Pull request closed without merging**